test/widgets/settings_test.dart

```diff
@@ -160,6 +160,24 @@ void main() {
     }, variant: TargetPlatformVariant({TargetPlatform.android, TargetPlatform.iOS}));
   });
 
+  group('SettingPageScrollBehavior', () {
+    testWidgets('scroll settings list when screen size is small', (tester) async {
```

Comment:
nit:

```suggestion
    testWidgets('content is scrollable when taller than a screenful', (tester) async {
```

(Helps focus the reader on what the test's purpose is, and acknowledges that the test can still be helpful when the screen is normal-sized, in a future where the list of settings is much longer.)